src/components/web-components/rank-card/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { StyleSheet } from 'react-native';
+import { ITheme } from '../../../core/theme/itheme';
+
+export default (theme: ITheme) =>
+    StyleSheet.create({
+        container: {
+            backgroundColor: theme.colors.rankCard,
+            marginRight: '16.84%',
```

Comment:
Use some consts here as much as possible - like BASE_DIMENSION in wallet app and adjust the values to something similar: like make a BASE_DIMENSION = 4. and for something like this you can use BASE_DIMENSION *4

Reply:
Okay. But when i do that, it's not responding to resizing the browser window

Reply:
Then maybe make a const as percentage. The idea is that this value will be almost everywhere the same margin